packages/rxjs-firebase/src/operators/querySnapshot.ts

```
@@ -1,21 +1,27 @@
 import { type DocumentData, type Query, type SnapshotListenOptions } from '@firebase/firestore'
-import { of, type OperatorFunction, startWith, switchMap } from 'rxjs'
+import { of, type OperatorFunction, startWith, Subject, switchMap, takeUntil, tap } from 'rxjs'
```

ESLint

[error] Unable to resolve path to module 'rxjs'.

(import-x/no-unresolved)